### PR TITLE
ci: make lint flake8 independent step

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

**Comment by JoanFM** [comment]
@hanxiao if accepted, how can I have the flake8 analysis as a required step?

**Comment by hanxiao** [comment]
why separate it from the main ci jobs?

**Comment by JoanFM** [comment]
> why separate it from the main ci jobs?

no big deal, to have only one and not one per version

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -52,6 +52,25 @@ jobs:
             This message will be deleted automatically when the commit messages get fixed.
           reaction-type: "eyes"
 
+  lint_flake_8:
```
Comment (hanxiao) [inline .github/workflows/ci.yml:55]:
```suggestion
  lint-flake-8:
```

**Review by hanxiao — APPROVED** [review]

**Pull request merged**